<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Carl Murie and Raphael Gottardo" />

<meta name="date" content="2018-08-09" />

<title>RTSS Project Report</title>

<script src="site_libs/jquery-1.12.4/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/lumen.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/htmlwidgets-1.2/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.4/datatables.js"></script>
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.16/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.16/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.0.0/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">RTSSProject</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/jdblischak/workflowr">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">RTSS Project Report</h1>
<h3 class="subtitle"><em>Presentation of vaccine and disease effects</em></h3>
<h4 class="author"><em>Carl Murie and Raphael Gottardo</em></h4>
<h4 class="date"><em>August 09, 2018</em></h4>

</div>


<p><strong>Last updated:</strong> 2018-08-09</p>
<strong>workflowr checks:</strong> <small>(Click a bullet for more information)</small>
<ul>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>R Markdown file:</strong> up-to-date </summary></p>
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Environment:</strong> empty </summary></p>
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Seed:</strong> <code>set.seed(20180706)</code> </summary></p>
<p>The command <code>set.seed(20180706)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Session information:</strong> recorded </summary></p>
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</details>
</li>
<li>
<p><details> <summary> <strong style="color:blue;">✔</strong> <strong>Repository version:</strong> 59a6a25 </summary></p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    analysis/.Rhistory

Untracked files:
    Untracked:  code/
    Untracked:  data/BTM_for_GSEA_20131008.gmt
    Untracked:  data/RawReadsFiltered.rds

Unstaged changes:
    Modified:   analysis/primaryAnalysisChildren/child_1_Stim.Rmd
    Modified:   analysis/primaryAnalysisChildren/child_2_age_DEG.Rmd
    Modified:   analysis/primaryAnalysisChildren/child_2_age_GSEA.Rmd

</code></pre>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes. </details>
</li>
</ul>
<details> <summary> <small><strong>Expand here to see past versions:</strong></small> </summary>
<ul>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
File
</th>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
<th style="text-align:left;">
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
59a6a25
</td>
<td style="text-align:left;">
<a href="mailto:carl.murie@gmail.com">carl.murie@gmail.com</a>
</td>
<td style="text-align:left;">
2018-08-09
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
8c023a8
</td>
<td style="text-align:left;">
<a href="mailto:carl.murie@gmail.com">carl.murie@gmail.com</a>
</td>
<td style="text-align:left;">
2018-08-09
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
386ac56
</td>
<td style="text-align:left;">
<a href="mailto:carl.murie@gmail.com">carl.murie@gmail.com</a>
</td>
<td style="text-align:left;">
2018-08-09
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
740e232
</td>
<td style="text-align:left;">
<a href="mailto:carl.murie@gmail.com">carl.murie@gmail.com</a>
</td>
<td style="text-align:left;">
2018-08-09
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
7f7464f
</td>
<td style="text-align:left;">
<a href="mailto:carl.murie@gmail.com">carl.murie@gmail.com</a>
</td>
<td style="text-align:left;">
2018-08-09
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
html
</td>
<td style="text-align:left;">
b1cb1e4
</td>
<td style="text-align:left;">
<a href="mailto:carl.murie@gmail.com">carl.murie@gmail.com</a>
</td>
<td style="text-align:left;">
2018-08-09
</td>
<td style="text-align:left;">
Build site.
</td>
</tr>
<tr>
<td style="text-align:left;">
Rmd
</td>
<td style="text-align:left;">
145404b
</td>
<td style="text-align:left;">
<a href="mailto:carl.murie@gmail.com">carl.murie@gmail.com</a>
</td>
<td style="text-align:left;">
2018-08-09
</td>
<td style="text-align:left;">
wflow_git_commit(c(“analysis/PrimaryAnalysis.Rmd”, “analysis/primaryAnalysisChildren/*“),
</td>
</tr>
</tbody>
</table>
</ul>
<p></details></p>
<hr />
<p></br></p>
<div id="statistical-analysis" class="section level1 smaller">
<h1>Statistical Analysis</h1>
<p>Investigate hbs and csp stimulations for M3 timepoint only.</p>
<ol style="list-style-type: decimal">
<li>Compare vaccine effects (rtss - comparator)<br />
</li>
<li>Compare disease effects (case - control)<br />
</li>
<li>Apply GSEA analysis (camera) with the Blood Transcriptional Modules (BTM) and individual gene testing (DEGs) to all comparisons
<ul>
<li>Apply comparisons to age groups both together and separately.</li>
<li>A Bayesian variance shrinkage estimator and the Benjamini-Hochberg FDR were applied.</li>
<li>An FDR cutoff of 0.2 was used for significance.</li>
</ul></li>
</ol>
<p></br></p>
</div>
<div id="gsea" class="section level1">
<h1>GSEA</h1>
<p></br></p>
<div id="hbs" class="section level2">
<h2><strong>HBS</strong></h2>
<p></br></p>
<div id="both-age-groups" class="section level3 tabset tabset-pills">
<h3>Both age groups</h3>
<p></br></p>
<p><strong>Sample sizes</strong></p>
<pre><code>       total BAGAMOYO MANHICA  F   M young old control case neither</code></pre>
<p>comparator 120 69 51 65 55 43 77 42 23 55 rtss 209 124 85 96 113 72 137 87 29 93</p>
<table>
<caption>disease samples</caption>
<thead>
<tr class="header">
<th></th>
<th align="center">total</th>
<th align="center">BAGAMOYO</th>
<th align="center">MANHICA</th>
<th align="center">F</th>
<th align="center">M</th>
<th align="center">young</th>
<th align="center">old</th>
<th align="center">comparator</th>
<th align="center">rtss</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>control</td>
<td align="center">87</td>
<td align="center">51</td>
<td align="center">36</td>
<td align="center">37</td>
<td align="center">50</td>
<td align="center">36</td>
<td align="center">51</td>
<td align="center">0</td>
<td align="center">87</td>
</tr>
<tr class="even">
<td>case</td>
<td align="center">29</td>
<td align="center">19</td>
<td align="center">10</td>
<td align="center">13</td>
<td align="center">16</td>
<td align="center">10</td>
<td align="center">19</td>
<td align="center">0</td>
<td align="center">29</td>
</tr>
</tbody>
</table>
<div id="vaccine-effects" class="section level4 unnumbered">
<h4><strong>Vaccine effects</strong></h4>
<div id="htmlwidget-a9c7df88dc0f768fdaed" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-a9c7df88dc0f768fdaed">{"x":{"filter":"none","caption":"<caption>Significant GSEA gene sets<\/caption>","data":[["enriched in NK cells (I) (M7.2)","signaling in T cells (II) (M35.1)","NK cell surface signature (S1)","signaling in T cells (I) (M35.0)","enriched in NK cells (II) (M61.0)","enriched in G-protein coupled receptors (M130)","enriched in T cells (I) (M7.0)","enriched in B cells (I) (M47.0)","CORO1A-DEF6 network (I) (M32.2)","MHC-TLR7-TLR8 cluster (M146)","mitotic cell cycle (M4.7)","plasma cells &amp; B cells, immunoglobulins (M156.0)","chromosome Y linked (M240)","T cell activation (II) (M7.3)","enriched in B cells (VI) (M69)","enriched in B cells (II) (M47.1)"],[45,10,24,13,13,10,56,39,17,16,11,17,9,29,18,32],["Up","Up","Up","Up","Up","Up","Up","Down","Down","Down","Down","Down","Up","Up","Down","Down"],[1.43e-11,7.32e-10,2.84e-08,1.22e-06,0.000213,0.000381,0.00118,0.00135,0.00137,0.00163,0.00214,0.00268,0.00312,0.00355,0.00562,0.00877],[3.8e-09,9.71e-08,2.51e-06,8.07e-05,0.0113,0.0168,0.0405,0.0405,0.0405,0.0432,0.0516,0.0593,0.0637,0.0671,0.0993,0.145]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="disease-effects" class="section level4 unnumbered">
<h4><strong>Disease effects</strong></h4>
<div id="htmlwidget-a211144b8093b472e4d6" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-a211144b8093b472e4d6">{"x":{"filter":"none","caption":"<caption>Significant GSEA gene sets<\/caption>","data":[["enriched in monocytes (I) (M4.15)"],[8],["Down"],[0.000396],[0.105]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="vaccine-effects---signficant-disease-gene-sets" class="section level4 unnumbered">
<h4><strong>Vaccine effects</strong> - signficant Disease gene sets</h4>
<div id="htmlwidget-7200922890551a11ff19" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-7200922890551a11ff19">{"x":{"filter":"none","caption":"<caption>Vaccine: signicant GSEA sets from disease<\/caption>","data":[["enriched in monocytes (I) (M4.15)"],[8],["Up"],[0.589],[0.589]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<p></br></p>
</div>
</div>
<div id="young-age-group" class="section level3 tabset tabset-pills">
<h3>Young age group</h3>
<p></br></p>
<p><strong>Sample sizes</strong></p>
<pre><code>       total BAGAMOYO MANHICA  F  M young old control case neither</code></pre>
<p>comparator 43 0 43 26 17 43 0 12 6 25 rtss 72 0 72 37 35 72 0 36 10 26</p>
<table>
<caption>disease samples</caption>
<thead>
<tr class="header">
<th></th>
<th align="center">total</th>
<th align="center">BAGAMOYO</th>
<th align="center">MANHICA</th>
<th align="center">F</th>
<th align="center">M</th>
<th align="center">young</th>
<th align="center">old</th>
<th align="center">comparator</th>
<th align="center">rtss</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>control</td>
<td align="center">36</td>
<td align="center">0</td>
<td align="center">36</td>
<td align="center">15</td>
<td align="center">21</td>
<td align="center">36</td>
<td align="center">0</td>
<td align="center">0</td>
<td align="center">36</td>
</tr>
<tr class="even">
<td>case</td>
<td align="center">10</td>
<td align="center">0</td>
<td align="center">10</td>
<td align="center">4</td>
<td align="center">6</td>
<td align="center">10</td>
<td align="center">0</td>
<td align="center">0</td>
<td align="center">10</td>
</tr>
</tbody>
</table>
<div id="vaccine-effects-1" class="section level4 unnumbered">
<h4><strong>Vaccine effects</strong></h4>
<div id="htmlwidget-de2b46a760547e92347d" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-de2b46a760547e92347d">{"x":{"filter":"none","caption":"<caption>Significant GSEA gene sets<\/caption>","data":[["enriched in B cells (I) (M47.0)","respiratory electron transport chain (mitochondrion) (M219)","TBA (M203)","cytoskeleton/actin (SRF transcription targets) (M145.1)","enriched in B cells (II) (M47.1)","enriched in B cells (VI) (M69)","respiratory electron transport chain (mitochondrion) (M238)","cytoskeleton/actin (SRF transcription targets) (M145.0)","KLF12 targets network (M32.3)"],[39,18,13,9,32,18,17,11,15],["Down","Down","Down","Up","Down","Down","Down","Up","Down"],[0.000114,0.000414,0.000605,0.000733,0.0018,0.00193,0.00302,0.00306,0.00446],[0.0301,0.0486,0.0486,0.0486,0.0853,0.0853,0.101,0.101,0.131]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="disease-effects-1" class="section level4 unnumbered">
<h4><strong>Disease effects</strong></h4>
<div id="htmlwidget-00ea80f5fe2175d5b247" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-00ea80f5fe2175d5b247">{"x":{"filter":"none","caption":"<caption>Significant GSEA gene sets<\/caption>","data":[["enriched in B cells (I) (M47.0)","MHC-TLR7-TLR8 cluster (M146)","enriched in B cells (VI) (M69)","respiratory electron transport chain (mitochondrion) (M231)","respiratory electron transport chain (mitochondrion) (M219)","antigen processing and presentation (M200)","enriched in B cells (II) (M47.1)","type I interferon response (M127)","proteasome (M226)","respiratory electron transport chain (mitochondrion) (M216)","transcription elongation, RNA polymerase II (M234)","viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)","respiratory electron transport chain (mitochondrion) (M238)","enriched in antigen presentation (III) (M95.1)","enriched in antigen presentation (II) (M95.0)","purine nucleotide biosynthesis (M212)","enriched in naive and memory B cells (M83)","chaperonin mediated protein folding (II) (M204.1)","TBA (source: B cells) (M152.0)","chaperonin mediated protein folding (I) (M204.0)","spliceosome (M250)","cell division - E2F transcription network (M4.8)","PLK1 signaling events (M4.2)","CORO1A-DEF6 network (I) (M32.2)","enriched in membrane proteins (M124)","myeloid cell cytokines, metallopeptidases and laminins (M78)","complement activation (I) (M112.0)","putative targets of PAX3 (M89.0)","enriched in antigen presentation (I) (M71)","translation initiation (M227)","TBA (M79)","regulation of antigen presentation and immune response (M5.0)","cell division (stimulated CD4+ T cells) (M46)","RIG-1 like receptor signaling (M68)","putative targets of PAX3 (M89.1)"],[39,16,18,11,18,6,32,12,12,12,13,11,17,13,21,12,9,10,6,14,12,14,20,17,13,9,10,12,16,10,7,76,13,10,9],["Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Up","Down","Down","Down","Up","Down","Up","Up","Up","Up","Down","Down","Up","Down","Up","Down","Up"],[9.36e-07,1.67e-06,9.21e-05,9.6e-05,0.000152,0.000161,0.000221,0.000279,0.000372,0.000627,0.000678,0.00102,0.00125,0.00129,0.00191,0.00226,0.00253,0.00254,0.00387,0.0046,0.00993,0.0109,0.014,0.0147,0.0154,0.017,0.0178,0.0182,0.0193,0.0195,0.022,0.0236,0.0246,0.0247,0.0254],[0.000221,0.000221,0.00636,0.00636,0.00713,0.00713,0.00835,0.00923,0.011,0.0163,0.0163,0.0224,0.0243,0.0243,0.0338,0.0373,0.0373,0.0373,0.0539,0.061,0.125,0.131,0.161,0.162,0.163,0.172,0.172,0.172,0.172,0.172,0.188,0.192,0.192,0.192,0.192]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="vaccine-effects---signficant-disease-gene-sets-1" class="section level4 unnumbered">
<h4><strong>Vaccine effects</strong> - signficant Disease gene sets</h4>
<div id="htmlwidget-629f8bfd6386c6d121a7" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-629f8bfd6386c6d121a7">{"x":{"filter":"none","caption":"<caption>Vaccine: signicant GSEA sets from disease<\/caption>","data":[["enriched in B cells (I) (M47.0)","MHC-TLR7-TLR8 cluster (M146)","enriched in B cells (VI) (M69)","respiratory electron transport chain (mitochondrion) (M231)","respiratory electron transport chain (mitochondrion) (M219)","antigen processing and presentation (M200)","enriched in B cells (II) (M47.1)","type I interferon response (M127)","proteasome (M226)","respiratory electron transport chain (mitochondrion) (M216)","transcription elongation, RNA polymerase II (M234)","viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)","respiratory electron transport chain (mitochondrion) (M238)","enriched in antigen presentation (III) (M95.1)","enriched in antigen presentation (II) (M95.0)","purine nucleotide biosynthesis (M212)","enriched in naive and memory B cells (M83)","chaperonin mediated protein folding (II) (M204.1)","TBA (source: B cells) (M152.0)","chaperonin mediated protein folding (I) (M204.0)","spliceosome (M250)","cell division - E2F transcription network (M4.8)","PLK1 signaling events (M4.2)","CORO1A-DEF6 network (I) (M32.2)","enriched in membrane proteins (M124)","myeloid cell cytokines, metallopeptidases and laminins (M78)","complement activation (I) (M112.0)","putative targets of PAX3 (M89.0)","enriched in antigen presentation (I) (M71)","translation initiation (M227)","TBA (M79)","regulation of antigen presentation and immune response (M5.0)","cell division (stimulated CD4+ T cells) (M46)","RIG-1 like receptor signaling (M68)","putative targets of PAX3 (M89.1)"],[39,16,18,11,18,6,32,12,12,12,13,11,17,13,21,12,9,10,6,14,12,14,20,17,13,9,10,12,16,10,7,76,13,10,9],["Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Up","Down","Down","Down","Up","Down","Down","Down","Up","Down","Up","Up","Up","Up","Down","Down","Up","Down","Up","Down","Up"],[0.000114,0.109,0.00193,0.0647,0.000414,0.0205,0.0018,0.0435,0.0501,0.0856,0.0138,0.0219,0.00302,0.569,0.815,0.473,0.118,0.409,0.0222,0.481,0.0318,0.327,0.514,0.011,0.0763,0.899,0.0964,0.939,0.133,0.202,0.798,0.601,0.413,0.0106,0.787],[0.00399,0.201,0.0169,0.151,0.00724,0.0706,0.0169,0.117,0.125,0.176,0.0604,0.0706,0.0211,0.687,0.864,0.624,0.206,0.578,0.0706,0.624,0.0928,0.498,0.643,0.055,0.167,0.925,0.187,0.939,0.222,0.321,0.864,0.701,0.578,0.055,0.864]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<p></br></p>
</div>
</div>
<div id="old-age-group" class="section level3 tabset tabset-pills">
<h3>Old age group</h3>
<p></br></p>
<p><strong>Sample sizes</strong></p>
<pre><code>       total BAGAMOYO MANHICA  F  M young old control case neither</code></pre>
<p>comparator 77 69 8 39 38 0 77 30 17 30 rtss 137 124 13 59 78 0 137 51 19 67</p>
<table>
<caption>disease samples</caption>
<thead>
<tr class="header">
<th></th>
<th align="center">total</th>
<th align="center">BAGAMOYO</th>
<th align="center">MANHICA</th>
<th align="center">F</th>
<th align="center">M</th>
<th align="center">young</th>
<th align="center">old</th>
<th align="center">comparator</th>
<th align="center">rtss</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>control</td>
<td align="center">51</td>
<td align="center">51</td>
<td align="center">0</td>
<td align="center">22</td>
<td align="center">29</td>
<td align="center">0</td>
<td align="center">51</td>
<td align="center">0</td>
<td align="center">51</td>
</tr>
<tr class="even">
<td>case</td>
<td align="center">19</td>
<td align="center">19</td>
<td align="center">0</td>
<td align="center">9</td>
<td align="center">10</td>
<td align="center">0</td>
<td align="center">19</td>
<td align="center">0</td>
<td align="center">19</td>
</tr>
</tbody>
</table>
<div id="vaccine-effects-2" class="section level4 unnumbered">
<h4><strong>Vaccine effects</strong></h4>
<div id="htmlwidget-03f0597c11ba9e09d898" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-03f0597c11ba9e09d898">{"x":{"filter":"none","caption":"<caption>Significant GSEA gene sets<\/caption>","data":[["enriched in NK cells (I) (M7.2)","signaling in T cells (II) (M35.1)","NK cell surface signature (S1)","signaling in T cells (I) (M35.0)","enriched in T cells (I) (M7.0)","TBA (M203)","enriched in G-protein coupled receptors (M130)","TBA (M72.2)","mitotic cell cycle (M4.7)","T cell differentiation via ITK and PKC (M18)","enriched in NK cells (II) (M61.0)","T cell activation (II) (M7.3)","enriched for ubiquitination (M138)","TBA (M105)","MHC-TLR7-TLR8 cluster (M146)","enriched in T cells (II) (M223)","T cell activation (III) (M7.4)","TBA (M72.1)","chemokine cluster (I) (M27.0)","TBA (M137)","plasma cells &amp; B cells, immunoglobulins (M156.0)"],[45,10,24,13,56,13,10,12,11,11,13,29,11,11,16,11,13,16,20,15,17],["Up","Up","Up","Up","Up","Up","Up","Up","Down","Up","Up","Up","Up","Down","Down","Up","Up","Up","Up","Up","Down"],[1.18e-13,1.38e-10,5.05e-07,8.61e-07,1.66e-06,1.84e-05,3.27e-05,0.000295,0.00049,0.000598,0.000778,0.001,0.00139,0.00159,0.00204,0.0104,0.0109,0.0109,0.0133,0.0151,0.0154],[3.12e-11,1.83e-08,4.46e-05,5.7e-05,8.79e-05,0.000812,0.00124,0.00976,0.0144,0.0159,0.0187,0.0222,0.0284,0.03,0.036,0.16,0.16,0.16,0.185,0.194,0.194]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="disease-effects-2" class="section level4 unnumbered">
<h4><strong>Disease effects</strong></h4>
<div id="htmlwidget-139b58426518930e882d" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-139b58426518930e882d">{"x":{"filter":"none","caption":"<caption>Significant GSEA gene sets<\/caption>","data":[["enriched in myeloid cells and monocytes (M81)","TBA (M188)","leukocyte differentiation (M160)"],[23,9,14],["Down","Down","Up"],[0.000401,0.00124,0.00156],[0.106,0.138,0.138]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="vaccine-effects---signficant-disease-gene-sets-2" class="section level4 unnumbered">
<h4><strong>Vaccine effects</strong> - signficant Disease gene sets</h4>
<div id="htmlwidget-cd6b97bc8cf095ebde03" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-cd6b97bc8cf095ebde03">{"x":{"filter":"none","caption":"<caption>Vaccine: signicant GSEA sets from disease<\/caption>","data":[["enriched in myeloid cells and monocytes (M81)","TBA (M188)","leukocyte differentiation (M160)"],[23,9,14],["Up","Down","Up"],[0.164,0.479,0.033],[0.246,0.479,0.099]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="summary" class="section level2">
<h2>Summary</h2>
<p>Summary of number of significant gene sets by age and stimulation.</p>
<p>allSummary 1 3 9 16 21 35 1 1 1 1 1 1 hbs csp both 16 1 young 9 35 old 21 3</p>
<p></br></p>
</div>
<div id="csp" class="section level2">
<h2><strong>CSP</strong></h2>
<p></br></p>
<div id="both-age-groups-1" class="section level3 tabset tabset-pills">
<h3>Both age groups</h3>
<p></br></p>
<p><strong>Sample sizes</strong></p>
<pre><code>       total BAGAMOYO MANHICA   F   M young old control case neither</code></pre>
<p>comparator 131 73 58 71 60 51 80 48 23 60 rtss 216 126 90 102 114 77 139 92 31 93</p>
<table>
<caption>disease samples</caption>
<thead>
<tr class="header">
<th></th>
<th align="center">total</th>
<th align="center">BAGAMOYO</th>
<th align="center">MANHICA</th>
<th align="center">F</th>
<th align="center">M</th>
<th align="center">young</th>
<th align="center">old</th>
<th align="center">comparator</th>
<th align="center">rtss</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>control</td>
<td align="center">92</td>
<td align="center">52</td>
<td align="center">40</td>
<td align="center">41</td>
<td align="center">51</td>
<td align="center">40</td>
<td align="center">52</td>
<td align="center">0</td>
<td align="center">92</td>
</tr>
<tr class="even">
<td>case</td>
<td align="center">31</td>
<td align="center">19</td>
<td align="center">12</td>
<td align="center">15</td>
<td align="center">16</td>
<td align="center">12</td>
<td align="center">19</td>
<td align="center">0</td>
<td align="center">31</td>
</tr>
</tbody>
</table>
<div id="vaccine-effects-3" class="section level4 unnumbered">
<h4><strong>Vaccine effects</strong></h4>
<div id="htmlwidget-1d1a3779f9007b9b5e14" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-1d1a3779f9007b9b5e14">{"x":{"filter":"none","caption":"<caption>Significant GSEA gene sets<\/caption>","data":[["enriched in NK cells (I) (M7.2)","signaling in T cells (II) (M35.1)","respiratory electron transport chain (mitochondrion) (M238)","spliceosome (M250)","enriched in T cells (I) (M7.0)","respiratory electron transport chain (mitochondrion) (M219)","signaling in T cells (I) (M35.0)","respiratory electron transport chain (mitochondrion) (M216)","proteasome (M226)","respiratory electron transport chain (mitochondrion) (M231)","TBA (M174)","enriched in NK cells (III) (M157)","TBA (M243)","intracellular transport (M147)","chaperonin mediated protein folding (II) (M204.1)","T cell differentiation (M14)","T cell differentiation via ITK and PKC (M18)","enriched in NK cells (II) (M61.0)","TBA (M72.0)","B cell surface signature (S2)","chaperonin mediated protein folding (I) (M204.0)","Naive B cell surface signature (S8)","T cell surface signature (S0)","AP-1 transcription factor network (M20)","cell cycle and growth arrest (M31)"],[45,10,17,12,56,18,13,12,12,11,22,13,12,17,10,12,11,13,21,70,14,24,25,13,11],["Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Down","Up","Down","Up","Up","Up"],[5.2e-07,2.49e-06,6.3e-06,1.43e-05,2.88e-05,6.93e-05,0.000135,0.000208,0.000243,0.000453,0.000458,0.00047,0.00108,0.00123,0.00141,0.00258,0.00481,0.00502,0.00583,0.0066,0.00882,0.0102,0.0109,0.0137,0.017],[0.000138,0.00033,0.000556,0.000949,0.00153,0.00306,0.00513,0.00689,0.00715,0.0104,0.0104,0.0104,0.022,0.0232,0.0249,0.0428,0.0739,0.0739,0.0813,0.0875,0.111,0.123,0.126,0.152,0.18]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="disease-effects-3" class="section level4 unnumbered">
<h4><strong>Disease effects</strong></h4>
<div id="htmlwidget-e964e474e774a8773efd" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-e964e474e774a8773efd">{"x":{"filter":"none","caption":"<caption>Significant GSEA gene sets<\/caption>","data":[["respiratory electron transport chain (mitochondrion) (M238)"],[17],["Down"],[0.000306],[0.0812]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="vaccine-effects---signficant-disease-gene-sets-3" class="section level4 unnumbered">
<h4><strong>Vaccine effects</strong> - signficant Disease gene sets</h4>
<div id="htmlwidget-5c4b9187591f96282d12" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-5c4b9187591f96282d12">{"x":{"filter":"none","caption":"<caption>Vaccine: signicant GSEA sets from disease<\/caption>","data":[["respiratory electron transport chain (mitochondrion) (M238)"],[17],["Up"],[6.3e-06],[6.3e-06]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<p></br></p>
</div>
</div>
<div id="young-age-group-1" class="section level3 tabset tabset-pills">
<h3>Young age group</h3>
<p></br></p>
<p><strong>Sample sizes</strong></p>
<pre><code>       total BAGAMOYO MANHICA  F  M young old control case neither</code></pre>
<p>comparator 51 0 51 30 21 51 0 16 6 29 rtss 77 0 77 41 36 77 0 40 12 25</p>
<table>
<caption>disease samples</caption>
<thead>
<tr class="header">
<th></th>
<th align="center">total</th>
<th align="center">BAGAMOYO</th>
<th align="center">MANHICA</th>
<th align="center">F</th>
<th align="center">M</th>
<th align="center">young</th>
<th align="center">old</th>
<th align="center">comparator</th>
<th align="center">rtss</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>control</td>
<td align="center">40</td>
<td align="center">0</td>
<td align="center">40</td>
<td align="center">18</td>
<td align="center">22</td>
<td align="center">40</td>
<td align="center">0</td>
<td align="center">0</td>
<td align="center">40</td>
</tr>
<tr class="even">
<td>case</td>
<td align="center">12</td>
<td align="center">0</td>
<td align="center">12</td>
<td align="center">6</td>
<td align="center">6</td>
<td align="center">12</td>
<td align="center">0</td>
<td align="center">0</td>
<td align="center">12</td>
</tr>
</tbody>
</table>
<div id="vaccine-effects-4" class="section level4 unnumbered">
<h4><strong>Vaccine effects</strong></h4>
<div id="htmlwidget-23e4355351a83b9a6fa7" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-23e4355351a83b9a6fa7">{"x":{"filter":"none","caption":"<caption>Significant GSEA gene sets<\/caption>","data":[["enriched in B cells (I) (M47.0)","transcription elongation, RNA polymerase II (M234)","TBA (source: B cells) (M152.0)","respiratory electron transport chain (mitochondrion) (M219)","translation initiation factor 3 complex (M245)","TBA (source: memory B cells) (M152.2)","transmembrane transport (I) (M87)","respiratory electron transport chain (mitochondrion) (M216)","E2F1 targets (Q3) (M10.0)","cytoskeleton/actin (SRF transcription targets) (M145.0)","cell division (stimulated CD4+ T cells) (M46)","enriched in B cells (VI) (M69)","enriched in naive and memory B cells (M83)","T cell activation (IV) (M52)","type I interferon response (M127)","enriched in B cells (II) (M47.1)","enriched in T cells (II) (M223)","enriched in monocytes (II) (M11.0)","TBA (M193)","nuclear pore complex (M106.0)","Monocyte surface signature (S4)","antigen processing and presentation (M200)","blood coagulation (M11.1)","enriched in neutrophils (I) (M37.1)","nuclear pore, transport; mRNA splicing, processing (M143)","cell cycle and transcription (M4.0)"],[39,13,6,18,11,6,19,12,23,11,13,18,9,13,12,32,11,146,10,17,69,6,22,29,11,226],["Down","Down","Up","Down","Down","Up","Up","Down","Down","Up","Up","Down","Down","Down","Down","Down","Down","Up","Up","Down","Up","Down","Up","Up","Down","Up"],[0.000165,0.00111,0.00248,0.00345,0.00366,0.00546,0.00577,0.00613,0.00626,0.00649,0.00666,0.00804,0.0084,0.00872,0.00879,0.00909,0.0102,0.011,0.0112,0.0132,0.014,0.0158,0.0164,0.0172,0.0178,0.0188],[0.0436,0.147,0.151,0.151,0.151,0.151,0.151,0.151,0.151,0.151,0.151,0.151,0.151,0.151,0.151,0.151,0.157,0.157,0.157,0.175,0.177,0.189,0.189,0.189,0.189,0.191]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="disease-effects-4" class="section level4 unnumbered">
<h4><strong>Disease effects</strong></h4>
<div id="htmlwidget-17f3ce3654c823c6b23e" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-17f3ce3654c823c6b23e">{"x":{"filter":"none","caption":"<caption>Significant GSEA gene sets<\/caption>","data":[["antigen processing and presentation (M200)","enriched in B cells (I) (M47.0)","B cell surface signature (S2)","enriched in B cells (II) (M47.1)","cell adhesion (GO) (M117)","proinflammatory cytokines and chemokines (M29)","enriched in dendritic cells (M168)","blood coagulation (M11.1)","cell cycle and growth arrest (M31)","viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)","putative targets of PAX3 (M89.1)","myeloid, dendritic cell activation via NFkB (I) (M43.0)","enriched in monocytes (I) (M4.15)","TBA (M194)","enriched in naive and memory B cells (M83)","Naive B cell surface signature (S8)","TBA (M72.0)","lysosomal/endosomal proteins (M139)","cytoskeletal remodeling (M32.8)","T cell signaling and costimulation (M44)","TBA (M66)","mitochondrial cluster (M235)","TBA (source: B cells) (M152.0)","leukocyte differentiation (M160)","enriched in B cells (VI) (M69)","inflammatory response (M33)","enriched in monocytes (II) (M11.0)","growth factor induced, enriched in nuclear receptor subfamily 4 (M94)","respiratory electron transport chain (mitochondrion) (M238)","TBA (M105)","plasma cells &amp; B cells, immunoglobulins (M156.0)","TBA (M114.0)","enriched in T cells (II) (M223)","TBA (M32.6)","regulation of signal transduction (M3)","T cell differentiation (Th2) (M19)","chemokines and receptors (M38)","type I interferon response (M127)","regulation of transcription, transcription factors (M213)"],[6,39,70,32,20,9,12,22,11,11,9,12,8,13,9,24,21,10,9,10,17,17,6,14,18,8,146,7,17,11,17,21,11,12,33,15,11,12,17],["Down","Down","Down","Down","Up","Up","Up","Up","Up","Down","Up","Up","Up","Up","Down","Down","Up","Up","Up","Up","Up","Down","Down","Up","Down","Up","Up","Up","Down","Up","Down","Up","Up","Up","Up","Up","Up","Down","Up"],[0.000177,0.000255,0.000259,0.000978,0.00161,0.0026,0.00292,0.00381,0.00397,0.00457,0.0058,0.00586,0.00631,0.00689,0.00695,0.00821,0.00826,0.00854,0.00868,0.00878,0.00919,0.0112,0.0116,0.0116,0.0121,0.0142,0.0156,0.0157,0.0157,0.016,0.0168,0.0186,0.0193,0.0196,0.0235,0.0248,0.0264,0.028,0.0293],[0.0229,0.0229,0.0229,0.0648,0.0855,0.11,0.11,0.116,0.116,0.116,0.116,0.116,0.116,0.116,0.116,0.116,0.116,0.116,0.116,0.116,0.116,0.128,0.128,0.128,0.128,0.142,0.142,0.142,0.142,0.142,0.143,0.152,0.152,0.152,0.178,0.182,0.189,0.195,0.199]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="vaccine-effects---signficant-disease-gene-sets-4" class="section level4 unnumbered">
<h4><strong>Vaccine effects</strong> - signficant Disease gene sets</h4>
<div id="htmlwidget-6ea8ebb028cf0ab773a3" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-6ea8ebb028cf0ab773a3">{"x":{"filter":"none","caption":"<caption>Vaccine: signicant GSEA sets from disease<\/caption>","data":[["antigen processing and presentation (M200)","enriched in B cells (I) (M47.0)","B cell surface signature (S2)","enriched in B cells (II) (M47.1)","cell adhesion (GO) (M117)","proinflammatory cytokines and chemokines (M29)","enriched in dendritic cells (M168)","blood coagulation (M11.1)","cell cycle and growth arrest (M31)","viral sensing &amp; immunity; IRF2 targets network (II) (M111.1)","putative targets of PAX3 (M89.1)","myeloid, dendritic cell activation via NFkB (I) (M43.0)","enriched in monocytes (I) (M4.15)","TBA (M194)","enriched in naive and memory B cells (M83)","Naive B cell surface signature (S8)","TBA (M72.0)","lysosomal/endosomal proteins (M139)","cytoskeletal remodeling (M32.8)","T cell signaling and costimulation (M44)","TBA (M66)","mitochondrial cluster (M235)","TBA (source: B cells) (M152.0)","leukocyte differentiation (M160)","enriched in B cells (VI) (M69)","inflammatory response (M33)","enriched in monocytes (II) (M11.0)","growth factor induced, enriched in nuclear receptor subfamily 4 (M94)","respiratory electron transport chain (mitochondrion) (M238)","TBA (M105)","plasma cells &amp; B cells, immunoglobulins (M156.0)","TBA (M114.0)","enriched in T cells (II) (M223)","TBA (M32.6)","regulation of signal transduction (M3)","T cell differentiation (Th2) (M19)","chemokines and receptors (M38)","type I interferon response (M127)","regulation of transcription, transcription factors (M213)"],[6,39,70,32,20,9,12,22,11,11,9,12,8,13,9,24,21,10,9,10,17,17,6,14,18,8,146,7,17,11,17,21,11,12,33,15,11,12,17],["Down","Down","Up","Down","Down","Down","Up","Up","Up","Down","Down","Down","Down","Up","Down","Down","Down","Up","Down","Down","Up","Down","Up","Down","Down","Up","Up","Up","Down","Up","Down","Up","Down","Up","Up","Down","Down","Down","Down"],[0.0158,0.000165,0.645,0.00909,0.253,0.877,0.156,0.0164,0.63,0.0237,0.563,0.692,0.613,0.615,0.0084,0.504,0.321,0.199,0.869,0.598,0.325,0.947,0.00248,0.174,0.00804,0.814,0.011,0.733,0.0273,0.0792,0.0588,0.146,0.0102,0.952,0.233,0.823,0.654,0.00879,0.0332],[0.064,0.00644,0.823,0.0536,0.47,0.924,0.358,0.064,0.823,0.084,0.823,0.843,0.823,0.823,0.0536,0.819,0.551,0.408,0.924,0.823,0.551,0.952,0.0484,0.377,0.0536,0.917,0.0536,0.866,0.0887,0.206,0.164,0.356,0.0536,0.952,0.454,0.917,0.823,0.0536,0.0996]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<p></br></p>
</div>
</div>
<div id="old-age-group-1" class="section level3 tabset tabset-pills">
<h3>Old age group</h3>
<p></br></p>
<p><strong>Sample sizes</strong></p>
<pre><code>       total BAGAMOYO MANHICA  F  M young old control case neither</code></pre>
<p>comparator 80 73 7 41 39 0 80 32 17 31 rtss 139 126 13 61 78 0 139 52 19 68</p>
<table>
<caption>disease samples</caption>
<thead>
<tr class="header">
<th></th>
<th align="center">total</th>
<th align="center">BAGAMOYO</th>
<th align="center">MANHICA</th>
<th align="center">F</th>
<th align="center">M</th>
<th align="center">young</th>
<th align="center">old</th>
<th align="center">comparator</th>
<th align="center">rtss</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>control</td>
<td align="center">52</td>
<td align="center">52</td>
<td align="center">0</td>
<td align="center">23</td>
<td align="center">29</td>
<td align="center">0</td>
<td align="center">52</td>
<td align="center">0</td>
<td align="center">52</td>
</tr>
<tr class="even">
<td>case</td>
<td align="center">19</td>
<td align="center">19</td>
<td align="center">0</td>
<td align="center">9</td>
<td align="center">10</td>
<td align="center">0</td>
<td align="center">19</td>
<td align="center">0</td>
<td align="center">19</td>
</tr>
</tbody>
</table>
<div id="vaccine-effects-5" class="section level4 unnumbered">
<h4><strong>Vaccine effects</strong></h4>
<div id="htmlwidget-b8d98d719a2a20c1212a" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-b8d98d719a2a20c1212a">{"x":{"filter":"none","caption":"<caption>Significant GSEA gene sets<\/caption>","data":[["enriched in NK cells (I) (M7.2)","respiratory electron transport chain (mitochondrion) (M238)","enriched in T cells (I) (M7.0)","respiratory electron transport chain (mitochondrion) (M219)","respiratory electron transport chain (mitochondrion) (M216)","TBA (M174)","spliceosome (M250)","signaling in T cells (II) (M35.1)","proteasome (M226)","respiratory electron transport chain (mitochondrion) (M231)","regulation of transcription, transcription factors (M213)","enriched in NK cells (III) (M157)","TBA (M72.0)","signaling in T cells (I) (M35.0)","TBA (M72.2)","enriched in T cells (II) (M223)","T cell surface signature (S0)","B cell surface signature (S2)","intracellular transport (M147)","cytoskeleton/actin (SRF transcription targets) (M145.0)","chaperonin mediated protein folding (II) (M204.1)","chemokine cluster (II) (M27.1)","TBA (M243)","TBA (M203)","TBA (M72.1)","transcription elongation, RNA polymerase II (M234)","TBA (M205)","T cell differentiation (M14)","T cell differentiation via ITK and PKC (M18)","translation initiation factor 3 complex (M245)","TBA (source: B cells) (M152.0)","mitotic cell cycle in stimulated CD4 T cells (M4.11)","AP-1 transcription factor network (M20)","chaperonin mediated protein folding (I) (M204.0)","small GTPase mediated signal transduction (M215)","TBA (M249)","cell cycle, ATP binding (M144)","cell adhesion (GO) (M117)","leukocyte differentiation (M160)","T cell activation (II) (M7.3)","TBA (M185)","enriched in NK cells (II) (M61.0)","chemokine cluster (I) (M27.0)","Naive B cell surface signature (S8)","DC surface signature (S5)","CD4 T cell surface signature Th2-stimulated (S7)","TBA (M128)","enriched in G-protein coupled receptors (M130)","cell cycle and growth arrest (M31)","T cell activation and signaling (M5.1)","enriched in B cells (V) (M47.4)","cell cycle (II) (M4.10)","T cell activation (IV) (M52)","TBA (M190)","TBA (source: memory B cells) (M152.2)","cytoskeleton/actin (SRF transcription targets) (M145.1)","T cell differentiation (Th2) (M19)","translation initiation (M227)","leukocyte migration (M88.0)","nuclear pore complex (mitosis) (M106.1)","leukocyte activation and migration (M45)","E2F transcription factor network (M8)","nuclear pore, transport; mRNA splicing, processing (M143)","golgi membrane (II) (M237)"],[45,17,56,18,12,22,12,10,12,11,17,13,21,13,12,11,25,70,17,11,10,13,12,13,16,13,11,12,11,11,6,6,13,14,16,6,16,20,14,29,14,13,20,24,54,9,11,10,11,22,11,7,13,10,6,9,15,10,12,11,11,7,11,17],["Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Down","Up","Down","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Down","Down","Up","Up","Up","Down","Up","Up","Up","Up","Up","Up","Up","Down","Down","Up","Up","Up","Up","Up","Up","Down","Up","Up","Down","Down","Up","Up","Down","Up","Up","Down","Up","Up"],[3.72e-08,4.47e-07,1.47e-06,2.56e-06,1.25e-05,5.91e-05,8.26e-05,0.000207,0.000479,0.000625,0.00065,0.000854,0.00127,0.00151,0.00151,0.00175,0.00188,0.00296,0.00351,0.0038,0.00403,0.00458,0.0057,0.00639,0.00659,0.00679,0.00911,0.00991,0.0104,0.0113,0.0121,0.0122,0.0133,0.0149,0.0158,0.0178,0.0181,0.0182,0.0194,0.0218,0.0222,0.0222,0.0223,0.0251,0.026,0.0266,0.0282,0.0298,0.0302,0.0304,0.0319,0.0324,0.0325,0.0349,0.035,0.0361,0.0395,0.0406,0.0417,0.0425,0.0436,0.0443,0.0444,0.0463],[9.85e-06,5.92e-05,0.00013,0.000169,0.00066,0.00261,0.00313,0.00684,0.0141,0.0156,0.0156,0.0189,0.0258,0.0266,0.0266,0.029,0.0293,0.0435,0.0489,0.0503,0.0508,0.0552,0.0656,0.0692,0.0692,0.0692,0.0894,0.0938,0.095,0.1,0.101,0.101,0.106,0.116,0.119,0.127,0.127,0.127,0.132,0.137,0.137,0.137,0.137,0.151,0.153,0.153,0.159,0.161,0.161,0.161,0.163,0.163,0.163,0.168,0.168,0.171,0.184,0.185,0.187,0.187,0.187,0.187,0.187,0.192]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="disease-effects-5" class="section level4 unnumbered">
<h4><strong>Disease effects</strong></h4>
<div id="htmlwidget-4375996d8cd105c92486" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-4375996d8cd105c92486">{"x":{"filter":"none","caption":"<caption>Significant GSEA gene sets<\/caption>","data":[["T cell activation and signaling (M5.1)","enriched in T cells (I) (M7.0)","regulation of transcription, transcription factors (M213)","enriched in monocytes (I) (M4.15)","enriched in T cells (II) (M223)","translation initiation factor 3 complex (M245)","T cell differentiation via ITK and PKC (M18)","enriched in NK cells (III) (M157)","IL2, IL7, TCR network (M65)","TBA (M153)","respiratory electron transport chain (mitochondrion) (M216)","B cell surface signature (S2)","TBA (M32.5)","chemokine cluster (I) (M27.0)","T cell differentiation (M14)","respiratory electron transport chain (mitochondrion) (M238)","purine nucleotide biosynthesis (M212)"],[22,56,17,8,11,11,11,13,9,15,12,70,14,20,12,17,12],["Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Down","Up","Down","Down","Down","Down","Down"],[1.93e-05,5.2e-05,7.16e-05,0.00121,0.00172,0.00216,0.00331,0.00391,0.00431,0.00535,0.00671,0.00673,0.00792,0.0087,0.00896,0.00915,0.00952],[0.00512,0.00633,0.00633,0.0803,0.091,0.0954,0.125,0.127,0.127,0.142,0.148,0.148,0.148,0.148,0.148,0.148,0.148]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="vaccine-effects---signficant-disease-gene-sets-5" class="section level4 unnumbered">
<h4><strong>Vaccine effects</strong> - signficant Disease gene sets</h4>
<div id="htmlwidget-f81bf326806f42d09977" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-f81bf326806f42d09977">{"x":{"filter":"none","caption":"<caption>Vaccine: signicant GSEA sets from disease<\/caption>","data":[["T cell activation and signaling (M5.1)","enriched in T cells (I) (M7.0)","regulation of transcription, transcription factors (M213)","enriched in monocytes (I) (M4.15)","enriched in T cells (II) (M223)","translation initiation factor 3 complex (M245)","T cell differentiation via ITK and PKC (M18)","enriched in NK cells (III) (M157)","IL2, IL7, TCR network (M65)","TBA (M153)","respiratory electron transport chain (mitochondrion) (M216)","B cell surface signature (S2)","TBA (M32.5)","chemokine cluster (I) (M27.0)","T cell differentiation (M14)","respiratory electron transport chain (mitochondrion) (M238)","purine nucleotide biosynthesis (M212)"],[22,56,17,8,11,11,11,13,9,15,12,70,14,20,12,17,12],["Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Up","Down","Up","Up","Up","Up","Up"],[0.0304,1.47e-06,0.00065,0.474,0.00175,0.0113,0.0104,0.000854,0.154,0.0815,1.25e-05,0.00296,0.669,0.0223,0.00991,4.47e-07,0.284],[0.0431,1.25e-05,0.00276,0.504,0.00496,0.0192,0.0192,0.0029,0.187,0.107,7.08e-05,0.00719,0.669,0.0345,0.0192,7.6e-06,0.322]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>NGenes<\/th>\n      <th>Direction<\/th>\n      <th>PValue<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,3,4]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="summary-1" class="section level2">
<h2>Summary</h2>
<p>Summary of number of significant gene sets by age and stimulation.</p>
<p>allSummary 1 17 25 26 39 64 1 1 1 1 1 1 hbs csp both 25 1 young 26 39 old 64 17</p>
<p></br></p>
</div>
</div>
<div id="deg---differentially-expressed-genes" class="section level1">
<h1>DEG - differentially expressed genes</h1>
<p></br></p>
<div id="hbs-1" class="section level2">
<h2><strong>HBS</strong></h2>
<p></br></p>
<p></br></p>
<div id="both-age-groups-2" class="section level3 tabset tabset-pills">
<h3>Both age groups</h3>
<p></br></p>
<div id="vaccine-effects-6" class="section level4">
<h4><strong>Vaccine effects</strong></h4>
<p><img src="figure/PrimaryAnalysis.Rmd/unnamed-chunk-57-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of unnamed-chunk-57-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
b1cb1e4
</td>
<td style="text-align:left;">
<a href="mailto:carl.murie@gmail.com">carl.murie@gmail.com</a>
</td>
<td style="text-align:left;">
2018-08-09
</td>
</tr>
</tbody>
</table>
<p></details> <div id="htmlwidget-772944b4dc3a4ad10cc5" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-772944b4dc3a4ad10cc5">{"x":{"filter":"none","caption":"<caption>Significant DEG genes<\/caption>","data":[["IL2"],[0.738],[4.24e-08],[0.000482]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>log FC<\/th>\n      <th>P.value<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2,3]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script></p>
</div>
<div id="disease-effects-6" class="section level4">
<h4><strong>Disease effects</strong></h4>
<p><img src="figure/PrimaryAnalysis.Rmd/unnamed-chunk-58-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of unnamed-chunk-58-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
386ac56
</td>
<td style="text-align:left;">
<a href="mailto:carl.murie@gmail.com">carl.murie@gmail.com</a>
</td>
<td style="text-align:left;">
2018-08-09
</td>
</tr>
</tbody>
</table>
<p></details> <div id="htmlwidget-2dee4f0ea32a0439a4d5" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-2dee4f0ea32a0439a4d5">{"x":{"filter":"none","caption":"<caption>Significant DEG genes<\/caption>","data":[["EGR2"],[0.934],[6.65e-06],[0.0755]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>log FC<\/th>\n      <th>P.value<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2,3]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script></p>
</div>
<div id="vaccine-effects---signficant-disease-gene-sets-6" class="section level4">
<h4><strong>Vaccine effects</strong> - signficant Disease gene sets</h4>
<div id="htmlwidget-330f353ad69cf06d9b72" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-330f353ad69cf06d9b72">{"x":{"filter":"none","caption":"<caption>Vaccine: signicant GSEA sets from disease<\/caption>","data":[["EGR2"],[0.0613],[0.594],[0.594]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>log FC<\/th>\n      <th>P.value<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2,3]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<p></br></p>
</div>
</div>
<div id="young-age-group-2" class="section level3 tabset tabset-pills">
<h3>Young age group</h3>
<p></br></p>
<div id="vaccine-effects-7" class="section level4">
<h4><strong>Vaccine effects</strong></h4>
<p><img src="figure/PrimaryAnalysis.Rmd/unnamed-chunk-61-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of unnamed-chunk-61-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
b1cb1e4
</td>
<td style="text-align:left;">
<a href="mailto:carl.murie@gmail.com">carl.murie@gmail.com</a>
</td>
<td style="text-align:left;">
2018-08-09
</td>
</tr>
</tbody>
</table>
<p></details> No significant genes found</p>
</div>
<div id="disease-effects-7" class="section level4">
<h4><strong>Disease effects</strong></h4>
<p><img src="figure/PrimaryAnalysis.Rmd/unnamed-chunk-62-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of unnamed-chunk-62-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
8c023a8
</td>
<td style="text-align:left;">
<a href="mailto:carl.murie@gmail.com">carl.murie@gmail.com</a>
</td>
<td style="text-align:left;">
2018-08-09
</td>
</tr>
<tr>
<td style="text-align:left;">
386ac56
</td>
<td style="text-align:left;">
<a href="mailto:carl.murie@gmail.com">carl.murie@gmail.com</a>
</td>
<td style="text-align:left;">
2018-08-09
</td>
</tr>
</tbody>
</table>
<p></details></p>
<p>No significant genes found</p>
</div>
<div id="vaccine-effects---signficant-disease-gene-sets-7" class="section level4">
<h4><strong>Vaccine effects</strong> - signficant Disease gene sets</h4>
<p>No significant genes found</p>
<p></br></p>
</div>
</div>
<div id="old-age-group-2" class="section level3 tabset tabset-pills">
<h3>Old age group</h3>
<p></br></p>
<div id="vaccine-effects-8" class="section level4">
<h4><strong>Vaccine effects</strong></h4>
<p><img src="figure/PrimaryAnalysis.Rmd/unnamed-chunk-65-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of unnamed-chunk-65-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
386ac56
</td>
<td style="text-align:left;">
<a href="mailto:carl.murie@gmail.com">carl.murie@gmail.com</a>
</td>
<td style="text-align:left;">
2018-08-09
</td>
</tr>
<tr>
<td style="text-align:left;">
b1cb1e4
</td>
<td style="text-align:left;">
<a href="mailto:carl.murie@gmail.com">carl.murie@gmail.com</a>
</td>
<td style="text-align:left;">
2018-08-09
</td>
</tr>
</tbody>
</table>
<p></details> <div id="htmlwidget-edacb7ddb22deadbbb50" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-edacb7ddb22deadbbb50">{"x":{"filter":"none","caption":"<caption>Significant DEG genes<\/caption>","data":[["IL2"],[0.887],[8.3e-07],[0.00944]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>log FC<\/th>\n      <th>P.value<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2,3]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script></p>
</div>
<div id="disease-effects-8" class="section level4">
<h4><strong>Disease effects</strong></h4>
<p><img src="figure/PrimaryAnalysis.Rmd/unnamed-chunk-66-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of unnamed-chunk-66-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
8c023a8
</td>
<td style="text-align:left;">
<a href="mailto:carl.murie@gmail.com">carl.murie@gmail.com</a>
</td>
<td style="text-align:left;">
2018-08-09
</td>
</tr>
</tbody>
</table>
<p></details></p>
<p>No significant genes found</p>
</div>
<div id="vaccine-effects---signficant-disease-gene-sets-8" class="section level4">
<h4><strong>Vaccine effects</strong> - signficant Disease gene sets</h4>
<p>No significant genes found</p>
</div>
</div>
</div>
<div id="summary-2" class="section level2">
<h2>Summary</h2>
<p>Summary of number of significant gene sets by age and stimulation.</p>
<p>allSummary 0 1 3 3 hbs csp both 1 1 young 0 0 old 1 0</p>
<p></br></p>
</div>
<div id="csp-1" class="section level2">
<h2><strong>CSP</strong></h2>
<p></br></p>
<p></br></p>
<div id="both-age-groups-3" class="section level3 tabset tabset-pills">
<h3>Both age groups</h3>
<p></br></p>
<div id="vaccine-effects-9" class="section level4">
<h4><strong>Vaccine effects</strong></h4>
<p><img src="figure/PrimaryAnalysis.Rmd/unnamed-chunk-76-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of unnamed-chunk-76-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
b1cb1e4
</td>
<td style="text-align:left;">
<a href="mailto:carl.murie@gmail.com">carl.murie@gmail.com</a>
</td>
<td style="text-align:left;">
2018-08-09
</td>
</tr>
</tbody>
</table>
<p></details> <div id="htmlwidget-8537d74cada52581c865" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-8537d74cada52581c865">{"x":{"filter":"none","caption":"<caption>Significant DEG genes<\/caption>","data":[["IL2"],[0.476],[6.6e-07],[0.0075]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>log FC<\/th>\n      <th>P.value<\/th>\n      <th>FDR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2,3]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script></p>
</div>
<div id="disease-effects-9" class="section level4">
<h4><strong>Disease effects</strong></h4>
<p><img src="figure/PrimaryAnalysis.Rmd/unnamed-chunk-77-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of unnamed-chunk-77-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
8c023a8
</td>
<td style="text-align:left;">
<a href="mailto:carl.murie@gmail.com">carl.murie@gmail.com</a>
</td>
<td style="text-align:left;">
2018-08-09
</td>
</tr>
</tbody>
</table>
<p></details></p>
<p>No significant genes found</p>
</div>
<div id="vaccine-effects---signficant-disease-gene-sets-9" class="section level4">
<h4><strong>Vaccine effects</strong> - signficant Disease gene sets</h4>
<p>No significant genes found</p>
<p></br></p>
</div>
</div>
<div id="young-age-group-3" class="section level3 tabset tabset-pills">
<h3>Young age group</h3>
<p></br></p>
<div id="vaccine-effects-10" class="section level4">
<h4><strong>Vaccine effects</strong></h4>
<p><img src="figure/PrimaryAnalysis.Rmd/unnamed-chunk-80-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of unnamed-chunk-80-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
386ac56
</td>
<td style="text-align:left;">
<a href="mailto:carl.murie@gmail.com">carl.murie@gmail.com</a>
</td>
<td style="text-align:left;">
2018-08-09
</td>
</tr>
<tr>
<td style="text-align:left;">
b1cb1e4
</td>
<td style="text-align:left;">
<a href="mailto:carl.murie@gmail.com">carl.murie@gmail.com</a>
</td>
<td style="text-align:left;">
2018-08-09
</td>
</tr>
</tbody>
</table>
<p></details> No significant genes found</p>
</div>
<div id="disease-effects-10" class="section level4">
<h4><strong>Disease effects</strong></h4>
<p><img src="figure/PrimaryAnalysis.Rmd/unnamed-chunk-81-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of unnamed-chunk-81-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
8c023a8
</td>
<td style="text-align:left;">
<a href="mailto:carl.murie@gmail.com">carl.murie@gmail.com</a>
</td>
<td style="text-align:left;">
2018-08-09
</td>
</tr>
</tbody>
</table>
<p></details></p>
<p>No significant genes found</p>
</div>
<div id="vaccine-effects---signficant-disease-gene-sets-10" class="section level4">
<h4><strong>Vaccine effects</strong> - signficant Disease gene sets</h4>
<p>No significant genes found</p>
<p></br></p>
</div>
</div>
<div id="old-age-group-3" class="section level3 tabset tabset-pills">
<h3>Old age group</h3>
<p></br></p>
<div id="vaccine-effects-11" class="section level4">
<h4><strong>Vaccine effects</strong></h4>
<p><img src="figure/PrimaryAnalysis.Rmd/unnamed-chunk-84-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of unnamed-chunk-84-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
386ac56
</td>
<td style="text-align:left;">
<a href="mailto:carl.murie@gmail.com">carl.murie@gmail.com</a>
</td>
<td style="text-align:left;">
2018-08-09
</td>
</tr>
<tr>
<td style="text-align:left;">
b1cb1e4
</td>
<td style="text-align:left;">
<a href="mailto:carl.murie@gmail.com">carl.murie@gmail.com</a>
</td>
<td style="text-align:left;">
2018-08-09
</td>
</tr>
</tbody>
</table>
<p></details> No significant genes found</p>
</div>
<div id="disease-effects-11" class="section level4">
<h4><strong>Disease effects</strong></h4>
<p><img src="figure/PrimaryAnalysis.Rmd/unnamed-chunk-85-1.png" width="672" style="display: block; margin: auto;" /></p>
<details> <summary><em>Expand here to see past versions of unnamed-chunk-85-1.png:</em></summary>
<table style="border-collapse:separate; border-spacing:5px;">
<thead>
<tr>
<th style="text-align:left;">
Version
</th>
<th style="text-align:left;">
Author
</th>
<th style="text-align:left;">
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
8c023a8
</td>
<td style="text-align:left;">
<a href="mailto:carl.murie@gmail.com">carl.murie@gmail.com</a>
</td>
<td style="text-align:left;">
2018-08-09
</td>
</tr>
<tr>
<td style="text-align:left;">
386ac56
</td>
<td style="text-align:left;">
<a href="mailto:carl.murie@gmail.com">carl.murie@gmail.com</a>
</td>
<td style="text-align:left;">
2018-08-09
</td>
</tr>
</tbody>
</table>
<p></details></p>
<p>No significant genes found</p>
</div>
<div id="vaccine-effects---signficant-disease-gene-sets-11" class="section level4">
<h4><strong>Vaccine effects</strong> - signficant Disease gene sets</h4>
<p>No significant genes found</p>
</div>
</div>
</div>
<div id="summary-3" class="section level2">
<h2>Summary</h2>
<p>Summary of number of significant gene sets by age and stimulation.</p>
<p>allSummary 0 1 5 1 hbs csp both 1 0 young 0 0 old 0 0</p>
</div>
<div id="session-information" class="section level2">
<h2>Session information</h2>
<p>R version 3.4.4 (2018-03-15) Platform: x86_64-pc-linux-gnu (64-bit) Running under: Ubuntu 16.04.5 LTS</p>
<p>Matrix products: default BLAS: /usr/lib/libblas/libblas.so.3.6.0 LAPACK: /usr/lib/lapack/liblapack.so.3.6.0</p>
<p>locale: [1] LC_CTYPE=en_US.UTF-8 LC_NUMERIC=C<br />
[3] LC_TIME=en_US.UTF-8 LC_COLLATE=en_US.UTF-8<br />
[5] LC_MONETARY=en_US.UTF-8 LC_MESSAGES=en_US.UTF-8<br />
[7] LC_PAPER=en_US.UTF-8 LC_NAME=C<br />
[9] LC_ADDRESS=C LC_TELEPHONE=C<br />
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C</p>
<p>attached base packages: [1] stats4 parallel stats graphics grDevices utils datasets [8] methods base</p>
<p>other attached packages: [1] readr_1.1.1 gridExtra_2.3<br />
[3] ggplot2_3.0.0 xtable_1.8-2<br />
[5] DT_0.4 GSEABase_1.40.1<br />
[7] graph_1.56.0 annotate_1.56.2<br />
[9] XML_3.98-1.13 AnnotationDbi_1.40.0<br />
[11] IRanges_2.12.0 S4Vectors_0.16.0<br />
[13] edgeR_3.20.9 limma_3.34.9<br />
[15] Biobase_2.38.0 BiocGenerics_0.24.0<br />
[17] RNASeqUtilities_0.0.0.9000 knitr_1.20</p>
<p>loaded via a namespace (and not attached): [1] bit64_0.9-7 jsonlite_1.5 R.utils_2.6.0<br />
[4] here_0.1 shiny_1.1.0 assertthat_0.2.0 [7] highr_0.7 blob_1.1.1 yaml_2.2.0<br />
[10] pillar_1.3.0 RSQLite_2.1.1 backports_1.1.2<br />
[13] lattice_0.20-35 glue_1.3.0 digest_0.6.15<br />
[16] promises_1.0.1 colorspace_1.3-2 htmltools_0.3.6<br />
[19] httpuv_1.4.5 R.oo_1.22.0 plyr_1.8.4<br />
[22] pkgconfig_2.0.1 purrr_0.2.5 scales_0.5.0<br />
[25] whisker_0.3-2 later_0.7.3 git2r_0.23.0<br />
[28] tibble_1.4.2 withr_2.1.2 lazyeval_0.2.1<br />
[31] magrittr_1.5 crayon_1.3.4 mime_0.5<br />
[34] memoise_1.1.0 evaluate_0.11 R.methodsS3_1.7.1 [37] tools_3.4.4 hms_0.4.2 stringr_1.3.1<br />
[40] munsell_0.5.0 locfit_1.5-9.1 bindrcpp_0.2.2<br />
[43] compiler_3.4.4 rlang_0.2.1 grid_3.4.4<br />
[46] RCurl_1.95-4.11 htmlwidgets_1.2 crosstalk_1.0.0<br />
[49] labeling_0.3 bitops_1.0-6 rmarkdown_1.10<br />
[52] gtable_0.2.0 DBI_1.0.0 R6_2.2.2<br />
[55] dplyr_0.7.6 bit_1.1-14 bindr_0.1.1<br />
[58] workflowr_1.1.1 rprojroot_1.3-2 stringi_1.2.4<br />
[61] Rcpp_0.12.18 tidyselect_0.2.4</p>
</div>
</div>

<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>

<hr>
<p>
  This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a>
  analysis was created with
  <a href="https://github.com/jdblischak/workflowr">workflowr</a> 1.1.1
</p>
<hr>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
